### PR TITLE
Add grid-to-images command

### LINKED ISSUE #210: Limit one legend per image
Since the current capability is limited to mounting data on grids, it's best to limit one legend per image. Even if more than one simulation result is mounted on grids, only one is going to be visible.

### PR DESCRIPTION
**Pull request by devngc**
resolves #210

### CODE REVIEW

**Comment by devngc** [comment]
@mostaphaRoudsari, at this point, this repo is in need of a refactor and I am working on it as part of a 'continuous-refactoring' routine. https://github.com/orgs/ladybug-tools/projects/3/views/1

**Review by mostaphaRoudsari — COMMENTED** [review]
Hi @devngc! Overall looks good. I proposed two changes:

1. Re-use the `export-images` subcommand for exporting grid images.
2. Take the `grid-filter` key outside the config file.

File: honeybee_vtk/camera.py [hunk 1]
```diff
@@ -1,86 +1,155 @@
...
+                 parallel_scale: Union[int, None] = None) -> None:
 
         super().__init__(
             identifier=identifier, position=position, direction=direction,
-            up_vector=up_vector, h_size=h_size, v_size=v_size, type=type)
+            up_vector=up_vector, h_size=view_angle, type=projection)
+
+        self._view_angle = view_angle
+        self._projection = projection
+        self._reset_camera = reset_camera
```
[the remark this answers is not in the document]
Comment (devngc) [reply honeybee_vtk/camera.py:118]: I would keep this as is. _reset_camera is a bool. This value tells the renderer in the Assistant object to whether the reset the camera or not.

**Comment by devngc** [comment]
@mostaphaRoudsari,
I addressed the comments on [this](https://github.com/ladybug-tools/honeybee-vtk/pull/331) PR.

**Pull request closed without merging**